site2/docs/functions-develop.md

```diff
@@ -1006,7 +1006,22 @@ class MetricRecorderFunction(Function):
             context.record_metric('elevens-count', 1)
 ```
 <!--Go-->
-Currently, the feature is not available in Go.
+The Go SDK [`Context`](#context) object enables you to record metrics on a per-key basis. For example, you can set a metric for the `process-count` key and a different metric for the `elevens-count` key every time the function processes a message:
```

Comment:
@freeznet could you please help review this from the technical perspective? Thanks

Reply:
Worth noting that I more or less copied this from the Java one, since the method signature is the same and they use the same technique of overloading a single metric with labels.